scripts/soap_example.py

```diff
@@ -0,0 +1,173 @@
+
+import sys
+sys.path.insert(0,'../build/')
+import json
+import ase
+import argparse
+import rascal
+import rascal.lib as lrl
+import numpy as np
+from ase.io import read
+from rascal.representations import SOAP
+from rascal.utils import ostream_redirect
+def load_json(fn):
+    with open(fn,'r') as f:
+        data = json.load(f)
+    return data[str(data['ids'][0])]
+
+def json2ase(f):
+    return ase.Atoms(**{v:f[k] for k,v in
+dict(positions='positions',atom_types='numbers',pbc='pbc',cell='cell').items()
+})
+
+
+##########################################################################################
+##########################################################################################
+
+def get_spectrum(hypers, frames):
+    with ostream_redirect():
+        soap = SOAP(**hypers)
+        soap_vectors = soap.transform(frames)
+        spectrum = soap_vectors.get_feature_matrix()
+    return spectrum
+
+##########################################################################################
+
+def unravel_power_spectrum(spectrum, nspecies, ncen):
+    #rascal exploits the an <-> bn' symmetry without including multiplicty
+    #uravel the representation
+    spectrum = spectrum.reshape((ncen, int(nspecies*(nspecies+1)/2), -1))
+    y = np.zeros(tuple([ncen, nspecies, nspecies] + [i for i in spectrum.shape[2:]]))
+    counter = 0
+    for j in range(nspecies):
+        for k in range(j, nspecies):
+            y[:, j, k] = spectrum[:, counter]
+            y[:, k, j] = y[:, j, k]
+            counter += 1
+    y = y.reshape((ncen, -1))
+    return y
+
+##########################################################################################
+
+def normalise(spectrum):
+    x = spectrum
+    ncen = spectrum.shape[0]
+    for i in range(ncen):
+        norm = np.linalg.norm(x[i])
+        if norm >= 1.0e-20: x[i] /= norm
```

Comment:
Is this supposed to be some sort of `eps`-related value? Either way, it should be defined as a constant somewhere.